x/blobstream/types/validator.go

```diff
@@ -112,7 +113,8 @@ func EVMAddrLessThan(e common.Address, o common.Address) bool {
 // increases by 1% due to inflation, we shouldn't have to generate a new
 // validator set, after all the validators retained their relative percentages
 // during inflation and normalized Blobstream power shows no difference.
-func (ibv InternalBridgeValidators) PowerDiff(c InternalBridgeValidators) float64 {
+func (ibv InternalBridgeValidators) PowerDiff(
+	c InternalBridgeValidators) sdk.Dec {
 	powers := map[string]int64{}
```

Comment:
[note to the reviewers] Alternatively, we could perform all calculations using `sdk.Dec` from the outset. However, since there are no floating-point calculations involved until later in this function, I thought it might be more efficient to continue using integer operations until that point. Nevertheless, if it is preferable to use `sdk.Dec` arithmetic throughout, I can update the initial portion of the function accordingly.

Reply:
I think switching to `sdk.Dec` would be also future-proof in case we wanted to make any changes not to give to future devs the ability to choose. So, I would go with switching everything to `sdk.Dec`

Reply:
Thanks for your comment! 
I'm unsure how this change would effectively enforce any constraints on future developers when the function signature remains unchanged, and only its internals are to be modified to utilize `sdk.Dec`. Could you please provide more insight into your perspective?